_episodes/01-short-introduction-to-Python.md

```diff
@@ -444,16 +444,16 @@ for key in rev.keys():
 ~~~
 {: .language-python}
 ~~~
-1 -> one
-2 -> two
-3 -> three
+'first' -> one
+'second' -> two
+'third' -> three
 ~~~
 {: .output}
 
 > ## Changing dictionaries
 >
 > 1. First, print the value of the `rev` dictionary to the screen.
-> 2. Reassign the value that corresponds to the key `2` so that it no longer
+> 2. Reassign the value that corresponds to the key `second` so that it no longer
```

Comment:
In this new set of assignments, I do agree that using strings is a little more explicit. But I think we should make it clear that dictionaries are not only for strings. Perhaps we should have them reassign the value of `second` to the numeral `2` instead of apple-sauce.